astropy/timeseries/io/kepler.py

```diff
@@ -11,7 +11,7 @@
 __all__ = ["kepler_fits_reader"]
 
 
-def kepler_fits_reader(filename):
+def kepler_fits_reader(filename, unit_parse_strict="warn"):
```

Comment:
An alternative or addition might be to just have a generic `**kwargs` here that gets passed on. But probably overkill.

Reply:
Not sure all keywords would apply since this is a quite specific file format ?